### ISSUE TITLE
Is this tool compatible with Visual Studio Code?

### ISSUE THREAD

**Issue by Victor-aut** [ilionx/Test-Case-Association-Manager#28]

**Comment by wouterroos** [comment]
@tawwabDjalielie The DevOps extension doesn’t run in Visual Studio so that should work for any test assembly right?

**Comment by tawwabDjalielie** [comment]
@wouterroos that's correct. The extension does not use Visual Studio, nor does it use any Nuget packages related to it. Therefore, it would work if it is packaged  as a VS Code extension or maybe just run the exe via a custom powershell script. You can use the existing one for the Azure DevOps task as a base and tweak it to your likings:
https://github.com/ilionx/Test-Case-Association-Manager/blob/master/Marketplace%20Extension/associatetooltask/powershell.ps1. @Victor-aut feel free to do so!
